decidim-comments/app/frontend/application/apollo_client.ts

```diff
@@ -1,16 +1,20 @@
-import ApolloClient, { createNetworkInterface } from "apollo-client";
+import { InMemoryCache } from "apollo-cache-inmemory";
+import { ApolloClient } from "apollo-client";
+import { createHttpLink } from "apollo-link-http";
+
+import "unfetch/polyfill";
 
 // Create a custom network interface for Apollo since our
 // API endpoint is not the default.
-const networkInterface = createNetworkInterface({
+const httpLink = createHttpLink({
   uri: "/api",
-  opts: {
-    credentials: "same-origin",
-  },
+  fetch,
+  credentials: "same-origin",
 });
 
 const client = new ApolloClient({
-  networkInterface,
+  link: httpLink,
+  cache: new InMemoryCache(),
 });
 
```

Comment:
All this stuff is me following upgrade guides for `apollo-client` 2.x... But I don't really know what I'm doing :laughing: